README.md

```diff
@@ -1,6 +1,14 @@
 # Measure Repository
 
-A prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) and associated frontend application.
+A prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) and associated frontend application. This repository is a monorepo that consists of: 
+- [Measure Repository Service](https://github.com/projecttacoma/measure-repository/blob/main/service/README.md)
+  - Implements portions of the [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html) specification
+  - Acts as a FHIR server and shared source of truth for measures
```

Comment:
Is it a source of truth for just measures or measures and libraries? Or should we say artifacts?

Reply:
Hmm- the spec says "to support authoring, publishing, and distribution of FHIR-based quality measure specifications" ... maybe we say:
```suggestion
  - Acts as a FHIR server and shared source of truth for measures and associated artifacts
```
?